### PR TITLE
Feature/1110-1111 Schedules C1 and C2

### PR DESCRIPTION
**Pull request by Elaine-Krauss-TCG**
#1110
#1111

### CODE REVIEW

File: front-end/src/app/shared/models/schc1-transaction-type.model.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    memo_code: 'memo_code',
+    amount: 'loan_amount',
+    amountInputHeader: 'Loan Information',
+    aggregate: '',
+    purpose_description: '',
+    purposeDescripLabel: '',
+    text4000: 'text4000',
+    category_code: '',
+    election_code: 'election_code',
+    election_other_description: 'election_other_description',
```
Comment (mjtravers) [inline front-end/src/app/shared/models/schc1-transaction-type.model.ts]: I think election_code and election_other_description can be empty ''. I don't see those fields in the schedule C1 spec

maybe the same for memo_code and text400... we can add them back if they turn up on the input form. I'd rather remove them and have to add them later as if we leave them we might forget

File: front-end/src/app/shared/models/schc1-transaction.model.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  authorized_title: string | undefined;
+  @Transform(BaseModel.dateTransform) authorized_date_signed: Date | undefined;
+
+  entity_type: string | undefined;
+  receipt_line_number: string | undefined;
+  election_code: string | undefined;
+  election_other_description: string | undefined;
+  memo_code: boolean | undefined;
+  memo_text_description: string | undefined;
+  aggregation_group: AggregationGroups | undefined;
```
Comment (mjtravers) [inline front-end/src/app/shared/models/schc1-transaction.model.ts:54]: Where are the above fields (entity_type - aggregation_group) coming from?

File: front-end/src/app/shared/models/schc2-transaction.model.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  guarantor_occupation: string | undefined;
+  guaranteed_amount: number | undefined;
+
+  entity_type: string | undefined;
+  receipt_line_number: string | undefined;
+  election_code: string | undefined;
+  election_other_description: string | undefined;
+  memo_code: boolean | undefined;
+  memo_text_description: string | undefined;
+  aggregation_group: AggregationGroups | undefined;
```
Comment (mjtravers) [inline front-end/src/app/shared/models/schc2-transaction.model.ts:22]: Same question as C1 above about adding these fields.

File: front-end/src/app/shared/models/schc2-transaction-type.model.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    memo_code: 'memo_code',
+    amount: 'guaranteed_amount',
+    amountInputHeader: 'Loan Information',
+    aggregate: '',
+    purpose_description: '',
+    purposeDescripLabel: '',
+    text4000: 'text4000',
+    category_code: '',
+    election_code: 'election_code',
+    election_other_description: 'election_other_description',
```
Comment (mjtravers) [inline front-end/src/app/shared/models/schc2-transaction-type.model.ts]: Same comment as C1 above about defining values for fields not in the spec

**Review by mjtravers — COMMENTED** [review]
@Elaine-Krauss-TCG Have some questions about fields not in the schedule c1 or c2 spec but added to the models. Will check in with you to discuss

File: front-end/src/app/shared/models/schc1-transaction-type.model.ts (hunk 1 above)
Comment (Elaine-Krauss-TCG) [reply front-end/src/app/shared/models/schc1-transaction-type.model.ts]: I election_code, election_other_description, and memo_code to empty.

I kept text4000, as I *expect* that that's necessary.

File: front-end/src/app/shared/models/schc2-transaction-type.model.ts (hunk 4 above)
Comment (Elaine-Krauss-TCG) [reply front-end/src/app/shared/models/schc2-transaction-type.model.ts]: Same response as C1 type model.

**Review by mjtravers — APPROVED** [review]

**Pull request merged**